src/megameklab/com/ui/supportVeh/SVEquipmentTab.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * MegaMekLab
+ * Copyright (c) 2021 - The MegaMek Team. All Rights Reserved.
+ *
+ * This program is  free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+package megameklab.com.ui.supportVeh;
+
+import megamek.common.Entity;
+import megamek.common.EquipmentType;
+import megamek.common.MiscType;
+import megamek.common.Mounted;
+import megameklab.com.ui.EntitySource;
+import megameklab.com.ui.generalUnit.AbstractEquipmentTab;
+import megameklab.com.ui.util.AbstractEquipmentDatabaseView;
+import megameklab.com.util.UnitUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+/*
+ * The Equipment Tab for Support Vehicles units showing the equipment database and the current loadout list.
+ *
+ * @author Simon (Juliez)
+ */
+class SVEquipmentTab extends AbstractEquipmentTab {
+
+    public SVEquipmentTab(EntitySource eSource) {
+        super(eSource);
+    }
+
+    @Override
+    protected AbstractEquipmentDatabaseView getEquipmentDatabaseView() {
+        return new SVEquipmentDatabaseView(eSource);
+    }
+
+    @Override
+    protected List<Mounted> getLoadout() {
+        List<Mounted> result = new ArrayList<>();
+        result.addAll(getEntity().getWeaponList());
+        result.addAll(getEntity().getAmmo());
+        result.addAll(getEntity().getMisc().stream().filter(this::showInLoadout).collect(toList()));
+        return result;
+    }
```

Comment:
Given this code seems to be repeated a bunch (BM, PM, SV... and those are the only three I've looked at yet), can this be moved to AbstractEquipmentTab and then the other calls override that?

Reply:
Yes, done.